flink-connectors/flink-connector-elasticsearch-base/src/test/java/org/apache/flink/connector/elasticsearch/table/ElasticsearchDynamicSinkFactoryBaseTest.java

```diff
@@ -227,4 +232,19 @@ public void validateWrongCredential() {
                                                 "")
                                         .build()));
     }
+
+    @Test
+    public void testSinkParallelism() {
+        ElasticsearchDynamicSinkFactoryBase sinkFactory = createSinkFactory();
+        DynamicTableSink sink =
+                sinkFactory.createDynamicTableSink(
+                        createPrefilledTestContext()
+                                .withOption(SINK_PARALLELISM.key(), "2")
+                                .build());
+        Assertions.assertTrue(sink instanceof ElasticsearchDynamicSink);
+        ElasticsearchDynamicSink esSink = (ElasticsearchDynamicSink) sink;
+        SinkProvider provider =
+                (SinkProvider) esSink.getSinkRuntimeProvider(new ElasticsearchUtil.MockContext());
+        Assertions.assertEquals(2, (long) provider.getParallelism().get());
```

Comment:
Can you please use assertJ for the assertions?

Reply:
I have used assertJ in this test case.